test/unit/widgets/dragAndDrop.test.ts

```diff
@@ -1,7 +1,11 @@
 import * as RdfLib from 'rdflib'
 import { JSDOM } from 'jsdom'
 
```

Comment:
Maybe good to push this through, however I will create another branch to finish this off.